docs/Actions/revoke-user-role.md

```diff
@@ -0,0 +1,197 @@
+---
+id: revoke-user-role
+title: Revoke User Role
+sidebar_label: Revoke User Role
+---
+
+> Audience: [`Business Users`](/docs/audience#business-users)<br/>
+> Skill Prerequisites: `Permission Scheme`
+
+## `Typical Use Cases`
+
+- Revoke roles of existing users
+- Remove access to resources for specific users
+
+## `Don't use it to`
+
+- Prevent users from accessing the system at all. Instead, mark users as deleted or unauthorize them
```

Comment:
Do we have actions for unauthorize and delete to mention them here?

Reply:
Yes. Mentioned the two actions now.